CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased] - 2024-08-02
```

Comment:
Why are there two unreleased sections, and why are they dated?

Reply:
I want to separate Larry's work from mine